app/app.js

```diff
@@ -149,22 +148,6 @@ window.run = function() {
             });
     };
 
-    PreviewController.init().then(function(previewController) {
```

Comment:
No worries! My fault for having Webpack build `app.js` to the wrong directory which is why we never caught this before :facepalm: